{
  "name": "Kubernetes Workload Vulnerability Findings",
  "description": "Provides a detailed list of individual vulnerability findings for every Kubernetes workload. Each row represents a unique vulnerability per package.",
  "owner": "example@example.com",
  "schedule": {
    "user": "example@example.com",
    "frequency": "ad-hoc"
  },
  "id": "434",
  "zones": [
    "sec-eng-tools-dev"
  ],
  "panels": [
    {
      "name": "Kubernetes Workload Vulnerability Findings",
      "type": "table",
      "description": "Table of all vulnerabilities found in Kubernetes workloads. Replacement for the legacy workload report",
      "filters": [
        {
          "name": "VulnSeverity",
          "op": "in",
          "values": [
            "Critical",
            "High",
            "Medium",
            "Low",
            "Negligible",
            "Other"
          ]
        },
        {
          "name": "StartTime",
          "op": "=",
          "values": "1752012000"
        },
        {
          "name": "EndTime",
          "op": "=",
          "values": "1752616800"
        }
      ],
      "data": [
        {
          "container_image": "defectdojo/defectdojo-django:latest",
          "container_image_distro": "debian 12.9",
          "container_image_id": "sha256:a520752f350909c191db45a598a88fcca2fa5db17a340dee6b3d0e36f4122e11",
          "kubernetes_cluster_name": "sec-eng-tools-dev",
          "kubernetes_namespace_name": "defectdojo",
          "kubernetes_pod_container_name": "celery",
          "kubernetes_workload_name": "defectdojo-celery-worker",
          "kubernetes_workload_type": "Deployment",
          "scan_package_in_use": false,
          "scan_package_name": "tar",
          "scan_package_path": "/path/path/file.abc",
          "scan_package_type": "OS",
          "scan_package_version": "1.34+dfsg-1.2+deb12u1",
          "vuln_cisa_kev_due_date": "",
          "vuln_cisa_kev_known_ransomware": "",
          "vuln_cisa_kev_publish_date": "",
          "vuln_cvss_score": 10,
          "vuln_cvss_vector": "AV:N/AC:L/Au:N/C:C/I:C/A:C",
          "vuln_cvss_version": "2.0",
          "vuln_disclosure_date": "2005-08-10T04:00:00Z",
          "vuln_fix_available": false,
          "vuln_fix_available_date": "1970-01-01T00:00:00Z",
          "vuln_fixed_in_version": "",
          "vuln_has_exploit": false,
          "vuln_id": "CVE-2005-2541",
          "vuln_risk_accepted": false,
          "vuln_severity": "Negligible"
        },
        {
          "container_image": "defectdojo/defectdojo-django:latest",
          "container_image_distro": "debian 12.9",
          "container_image_id": "sha256:a520752f350909c191db45a598a88fcca2fa5db17a340dee6b3d0e36f4122e11000000",
          "kubernetes_cluster_name": "sec-eng-tools-dev",
          "kubernetes_namespace_name": "defectdojo",
          "kubernetes_pod_container_name": "uwsgi",
          "kubernetes_workload_name": "defectdojo-django",
          "kubernetes_workload_type": "Deployment",
          "scan_package_in_use": false,
          "scan_package_name": "tar",
          "scan_package_path": "/path/path/file.abc",
          "scan_package_type": "OS",
          "scan_package_version": "1.34+dfsg-1.2+deb12u1",
          "vuln_cisa_kev_due_date": "",
          "vuln_cisa_kev_known_ransomware": "",
          "vuln_cisa_kev_publish_date": "",
          "vuln_cvss_score": 10,
          "vuln_cvss_vector": "AV:N/AC:L/Au:N/C:C/I:C/A:C",
          "vuln_cvss_version": "2.0",
          "vuln_disclosure_date": "2005-08-10T04:00:00Z",
          "vuln_fix_available": false,
          "vuln_fix_available_date": "1970-01-01T00:00:00Z",
          "vuln_fixed_in_version": "",
          "vuln_has_exploit": false,
          "vuln_id": "CVE-2005-2541",
          "vuln_risk_accepted": false,
          "vuln_severity": "Negligible"
        },
        {
          "container_image": "defectdojo/defectdojo-django:latest",
          "container_image_distro": "debian 12.9",
          "container_image_id": "sha256:a520752f350909c191db45a598a88fcca2fa5db17a340dee6b3d0e36f4122e11",
          "kubernetes_cluster_name": "sec-eng-tools-dev",
          "kubernetes_namespace_name": "defectdojo",
          "kubernetes_pod_container_name": "celery",
          "kubernetes_workload_name": "defectdojo-celery-beat",
          "kubernetes_workload_type": "Deployment",
          "scan_package_in_use": false,
          "scan_package_name": "tar",
          "scan_package_path": "/path/path/file.abc",
          "scan_package_type": "OS",
          "scan_package_version": "1.34+dfsg-1.2+deb12u1",
          "vuln_cisa_kev_due_date": "",
          "vuln_cisa_kev_known_ransomware": "",
          "vuln_cisa_kev_publish_date": "",
          "vuln_cvss_score": 10,
          "vuln_cvss_vector": "AV:N/AC:L/Au:N/C:C/I:C/A:C",
          "vuln_cvss_version": "2.0",
          "vuln_disclosure_date": "2005-08-10T04:00:00Z",
          "vuln_fix_available": false,
          "vuln_fix_available_date": "1970-01-01T00:00:00Z",
          "vuln_fixed_in_version": "",
          "vuln_has_exploit": false,
          "vuln_id": "CVE-2005-2541",
          "vuln_risk_accepted": false,
          "vuln_severity": "Negligible"
        },
        {
          "container_image": "docker.io/bitnami/redis:7.2.5-debian-12-r4",
          "container_image_distro": "debian 12.6",
          "container_image_id": "sha256:a520752f350909c191db45a598a88fcca2fa5db17a340dee6b3d0e36f4122e11",
          "kubernetes_cluster_name": "sec-eng-tools-dev",
          "kubernetes_namespace_name": "defectdojo",
          "kubernetes_pod_container_name": "redis",
          "kubernetes_workload_name": "defectdojo-redis-master",
          "kubernetes_workload_type": "StatefulSet",
          "scan_package_in_use": false,
          "scan_package_name": "tar",
          "scan_package_path": "/path/path/file.abc",
          "scan_package_type": "OS",
          "scan_package_version": "1.34+dfsg-1.2+deb12u1",
          "vuln_cisa_kev_due_date": "",
          "vuln_cisa_kev_known_ransomware": "",
          "vuln_cisa_kev_publish_date": "",
          "vuln_cvss_score": 10,
          "vuln_cvss_vector": "AV:N/AC:L/Au:N/C:C/I:C/A:C",
          "vuln_cvss_version": "2.0",
          "vuln_disclosure_date": "2005-08-10T04:00:00Z",
          "vuln_fix_available": false,
          "vuln_fix_available_date": "1970-01-01T00:00:00Z",
          "vuln_fixed_in_version": "",
          "vuln_has_exploit": false,
          "vuln_id": "CVE-2005-2541",
          "vuln_risk_accepted": false,
          "vuln_severity": "Negligible"
        },
        {
          "container_image": "defectdojo/defectdojo-django:latest",
          "container_image_distro": "debian 12.9",
          "container_image_id": "sha256:a520752f350909c191db45a598a88fcca2fa5db17a340dee6b3d0e36f4122e11",
          "kubernetes_cluster_name": "sec-eng-tools-dev",
          "kubernetes_namespace_name": "defectdojo",
          "kubernetes_pod_container_name": "celery",
          "kubernetes_workload_name": "defectdojo-celery-beat",
          "kubernetes_workload_type": "Deployment",
          "scan_package_in_use": true,
          "scan_package_name": "pillow",
          "scan_package_path": "/path/path/path/file.abc",
          "scan_package_type": "Python",
          "scan_package_version": "11.2.1",
          "vuln_cisa_kev_due_date": "",
          "vuln_cisa_kev_known_ransomware": "",
          "vuln_cisa_kev_publish_date": "",
          "vuln_cvss_score": 9.8,
          "vuln_cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
          "vuln_cvss_version": "3.0",
          "vuln_disclosure_date": "2025-07-01T17:29:37Z",
          "vuln_fix_available": true,
          "vuln_fix_available_date": "2025-07-01T09:15:58Z",
          "vuln_fixed_in_version": "11.3.0",
          "vuln_has_exploit": false,
          "vuln_id": "CVE-2025-48379",
          "vuln_risk_accepted": false,
          "vuln_severity": "Other"
        },
        {
          "container_image": "defectdojo/defectdojo-django:latest",
          "container_image_distro": "debian 12.9",
          "container_image_id": "sha256:a520752f350909c191db45a598a88fcca2fa5db17a340dee6b3d0e36f4122e11",
          "kubernetes_cluster_name": "sec-eng-tools-dev",
          "kubernetes_namespace_name": "defectdojo",
          "kubernetes_pod_container_name": "uwsgi",
          "kubernetes_workload_name": "defectdojo-django",
          "kubernetes_workload_type": "Deployment",
          "scan_package_in_use": false,
          "scan_package_name": "libxml2",
          "scan_package_path": "/path/path/file.abc",
          "scan_package_type": "OS",
          "scan_package_version": "2.9.14+dfsg-1.3~deb12u1",
          "vuln_cisa_kev_due_date": "",
          "vuln_cisa_kev_known_ransomware": "",
          "vuln_cisa_kev_publish_date": "",
          "vuln_cvss_score": 0,
          "vuln_cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N",
          "vuln_cvss_version": "3.0",
          "vuln_disclosure_date": "1999-01-01T00:00:00Z",
          "vuln_fix_available": true,
          "vuln_fix_available_date": "1970-01-01T00:00:00Z",
          "vuln_fixed_in_version": "2.9.14+dfsg-1.3~deb12u2",
          "vuln_has_exploit": false,
          "vuln_id": "CVE-2023-45322",
          "vuln_risk_accepted": false,
          "vuln_severity": "Negligible"
        }
      ]
    }
  ]
}
